setup.py

```diff
@@ -69,7 +71,18 @@ def finalize_options(self):
         pass
 
     def run(self):
-        os.system('rm -vrf ./build ./dist ./*.pyc ./*.tgz ./*.egg-info')
+        # TODO: Review me! It might be a good idea to run tests in docker but maybe it's not
```

Comment:
I think not - if people (i.e. me!) still want to run some tests locally I think we should allow it

Reply:
yeah